src/main/java/view_post/use_case/ViewPostResponseModel.java

```diff
@@ -0,0 +1,64 @@
+package view_post.use_case;
+
+/**
+ * A data class for storing the output data.
+ */
+public class ViewPostResponseModel {
+    private final String posterEmail;
+    private final String postBody;
+    private final String[] postTags;
```

Comment:
can we have postTags be a String instead?

Reply:
Will do